src/app/component/general/eco-news/create-news/create-news.component.css

```diff
@@ -27,6 +37,14 @@
     align-items: center;
 }
 
+input.ng-invalid.ng-touched, textarea.ng-invalid.ng-touched {
+    border: 1px solid red !important;
+}
+
+input.ng-invalid.ng-touched + span, textarea.ng-invalid.ng-touched + p {
+    color: red;
+}
+
 .title h2 {
     font-size: 48px;
     font-family: 'Montserrat', sans-serif;
```

Comment:
Please rename 'info' class, 'cause it's not clear

Reply:
fixed